src/framework/tooltip/_tooltip.scss

```diff
@@ -0,0 +1,50 @@
+
+$arrowSide: 6px;
+$bgColor: #fafdfe;
+$borderColor: #dbe2e3;
+
+.tooltip {
+  padding: 15px 13px;
+}
+
+.tooltipWrapper {
+  display: inline-block;
+  position: relative;
+}
+
+.tooltipContainer {
+  display: none;
+  opacity: 0;
+  position: absolute;
+  z-index: 10;
+
+  &:after {
+    background-color: $bgColor;
+    border: solid $borderColor;
+    border-width: 1px 0 0 1px;
+    content: '';
+    display: block;
+    height: $arrowSide;
+    position: absolute;
+    margin-left: -$arrowSide;
+    margin-top: -4px;
```

Comment:
At least on chrome, `margin-top` should be 3. Otherwise there is an overlap of borders.